staging/src/k8s.io/apiextensions-apiserver/hack/build-image.sh

```diff
@@ -24,7 +24,7 @@ function cleanup {
 }
 trap cleanup EXIT
 
-pushd "${KUBE_ROOT}/vendor/k8s.io/apiextensions-apiserver"
+pushd "${KUBE_ROOT}/vendor/k8s.io/apiextensions-apiserver" || exit
```

Comment:
this should be a `exit 1`

Reply:
put `set -o errexit` at the top of this file (and the other options, like most of our scripts do

Reply:
Remove the `|| exit`, otherwise it won't fail

Reply:
thanks, removed it and also passed shellcheck. Cool :)